rootfs/api/models/build.py

```diff
@@ -43,10 +43,11 @@ def create(self, user, *args, **kwargs):
         try:
             self.app.deploy(new_release)
             return new_release
-        except Exception:
+        except Exception as e:
             if 'new_release' in locals():
                 new_release.delete()
-            raise
+
+            raise DeisException(str(e)) from e
```

Comment:
`DeisException` is our generic, like `EnvironmentError` was before - We can always introduce more Exception classes if we want to expose different things up to the API View or generally keep track of things differently

One thing that comes to mind for expansion: `KubeServiceUnavailable` that extends `ServiceUnavailable` and same way `LoggerServiceUnavailable` - Ultimately boils down to a `503` in the View.